### ISSUE TITLE
Very large values (2,937,379,629,571,585.00) in pidstat cpu_usage for two processes /opt/cni/bin/openshift-sdn

### ISSUE THREAD

**Issue by wabouhamad** [distributed-system-analysis/pbench#799]
While running pbench-ansible from a pbench-controller host on an OCP 3.9 cluster, pidstat cpu_usage on one of the compute nodes "svt_node_4:ip-172-31-55-116" shows two processes /opt/cni/bin/openshift-sdn with a huge value for average: 2,937,379,629,571,585.00.

The csv data shows several processes for /opt/cni/bin/openshift-sdn and they to all seem to have zero values. 

This appears to be a pidstat posprocessing issue.

The data are on the pbench server in the following directory:
<server_results_main_url>/EC2::ip-172-31-40-62/ocp-3.9.14-rpm-docker1.13.1-500ppn-pausepods-kern-310-861-m4XL_160/tools-default/svt_node_4:ip-172-31-55-116/pidstat/cpu_usage.html

The second compute node does not show this:
<server_results_main_url>/results/EC2::ip-172-31-40-62/ocp-3.9.14-rpm-docker1.13.1-500ppn-pausepods-kern-310-861-m4XL_160/tools-default/svt_node_3:ip-172-31-24-150/pidstat/cpu_usage.html

**Comment by ndokos** [comment]
Here's a link: http://pbench.perf.lab.eng.bos.redhat.com/results/EC2::ip-172-31-40-62/ocp-3.9.14-rpm-docker1.13.1-500ppn-pausepods-kern-310-861-m4XL_160/tools-default/svt_node_4:ip-172-31-55-116/pidstat/

Actually, the csv data also show a (different) large value: line 582, column 3144:1844674407370955264.00.

**Comment by sjug** [comment]
What do I win?

**Comment by ndokos** [comment]
You get to fix pidstat.

**Comment by ndokos** [comment]
Yes, that is the issue. Not sure what you mean in the second question?

**Comment by jeremyeder** [comment]
there is something systemically wrong - we've seen this in tools like iostat, too.  it could even be a kernel issue.
